src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcement.tsx

```diff
@@ -34,22 +45,42 @@ export const Announcement = ({announcement}: Props) => {
           {getTextForLanguage(announcement.summary, language)}
         </ThemeText>
       </View>
+      <PressableOpacity
+        style={style.close}
+        role="button"
+        hitSlop={12}
+        accessibilityHint={t(
+          DashboardTexts.announcemens.announcement.closeA11yHint,
+        )}
+        onPress={() => onDismiss(announcement)}
+      >
+        <ThemeIcon size={closeIconSize} svg={Close} />
+      </PressableOpacity>
```

Comment:
Could we align the implementation of the dismiss button in this component and in `MessageBox`? Now one is done with absolute positioning while the other is done without. I don't have a preference of which implementation to use, just that I think it should be the same so the screen reader experience is equal between them.

Reply:
I'll have a look

Reply:
The `MessageBox` has two columns, where one contains the text and the other the close icon. This approach is very safe in terms of text overlapping the icon, but it also means that longer messages will appear to have an unnaturally large right margin. 

The absolute positioning approach makes long texts look more natural, but there is more risk of text overlapping the icon. Absolute positioning works as long as we know that the height of the icon is approximately the same height as the heading line height, and when we know that the heading is required. The `marginRight` set on the header prevents this from overflowing the icon.

There is no "best" solution here. Both have tradeoffs, as described. I'm not sure if there's even any point in using the same approach in both places. What's your opinion, @gorandalum ?

Reply:
Good summary. Since the heading is required on announcement it makes sense that it can have absolute position, so I'm ok with different implementations. Just need to be sure that screen reader focus order is correct both on iOS and Android.

Reply:
I've done a few tests now and the difference is not that big. Since the text is not justified the line wrapping will often lead to large right margins anyway, so the benefit of the absolute approach is not that great. For the sake of safety and consistency I'll use the same approach as `MessageBox`.